.github/workflows/post-apiview.yml

```
@@ -0,0 +1,29 @@
+name: After APIView
+
+on:
+  check_run:
+    types: [completed]
+    
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  event-handler:
```

```suggestion
  post-apiview:
    name: After APIView
```

Convention is to use same name for workflow and job.